src/main/resources/swagger/api-docs.yaml

```diff
@@ -221,6 +221,51 @@ paths:
             - profile
 
   '/api/cluster/{googleProject}/{clusterName}':
+    patch:
```

Comment:
Maybe not for this PR but I'm wondering if we should consider retroactively versioning all our endpoints. E.g. `PATCH /api/cluster/v1/{googleProject/{clusterName}`. We have a v2 endpoint but no v1s. Maybe as a separate PR we could do a one time migration to make all endpoints versioned.

Reply:
Wouldn't retroactive versioning break clients, which defeats the purpose of versioning for backwards-compatibility, or did you imply the non-versioned endpoints would alias to v1? 

If we'll version our endpoints going forward, it'd be start versioning this one, too while at it.

Reply:
aliasing non-versioned routes to v1 is how we do it in sam. i agree that we should do this in a follow-up PR

Reply:
To be clear, what I'd suggested doing now was to make the endpoint route `/api/cluster/v1/{googleProject/{clusterName}` to begin with, to save the aliasing later on.

Reply:
if we do it like how it was done in sam, we can blanket-alias across all of the routes so it would be more work to version this endpoint now for little gain, IMO

Reply:
oh okay...I agree then